### PR TITLE
Model and Relationship

### PR DESCRIPTION
**Pull request by annq3012**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-0
- Task ID: #0.3

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Model/Book.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'category_id',
+        'name',
+        'author',
+        'year',
+        'description',
+        'donate_by',
```
Comment (at-anhngo) [inline app/Model/Book.php]: missing field price

**Comment by at-annguyen** [comment]
library_books

File: app/Model/Comment.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Comment extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'comment';
```
Comment (at-duyho) [inline app/Model/Comment.php]: - table name have to a plural noun

File: app/Model/Book.php [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Book extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'book';
```
Comment (at-duyho) [inline app/Model/Book.php]: - table name have to a plural noun

File: app/Model/Borrow.php [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Borrow extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'borrow';
```
Comment (at-duyho) [inline app/Model/Borrow.php]: - table name have to a plural noun

File: app/Model/Category.php [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'category';
```
Comment (at-duyho) [inline app/Model/Category.php]: - table name have to a plural noun

File: app/Model/Donator.php [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Donator extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'donator';
```
Comment (at-duyho) [inline app/Model/Donator.php]: - table name have to a plural noun

File: app/Model/Like.php [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class LikeAndShare extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'like_and_share';
```
Comment (at-duyho) [inline app/Model/Like.php]: - table name have to a plural noun

File: app/Model/Post.php [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'post';
```
Comment (at-duyho) [inline app/Model/Post.php]: - table name have to a plural noun

File: app/Model/Rating.php [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Rating extends Model
+{
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'rating';
```
Comment (at-duyho) [inline app/Model/Rating.php]: - table name have to a plural noun

**Review by ghost — CHANGES_REQUESTED** [review]
please check my comment. thank you

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**